src/libpgmoneta/backup.c

```diff
@@ -235,8 +236,10 @@ pgmoneta_backup(int client_fd, int server, uint8_t compression, uint8_t encrypti
    }
 
    size = pgmoneta_directory_size(d);
-   pgmoneta_update_info_unsigned_long(root, INFO_BACKUP, size);
-
+   pgmoneta_get_backup(root, NULL, &temp_backup);
+   temp_backup->backup_size = size;
+   pgmoneta_save_info(root, NULL, temp_backup);
```

Comment:
Would be nice if we can write changes to an in memory struct instance and apply them to the backup directly (empty fields will be ignored). But I think it doesn't have to go into this PR.